api/src/main/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimController.java

```diff
@@ -69,6 +72,16 @@ public ClaimDTO retrieveClaimById(@PathVariable("id") UUID id) {
         return claimToClaimDTOConverter.convert(claim);
     }
 
+    @GetMapping
```

Comment:
please bare in mind if you are changing this to **PostMapping** there then you may need to add path (/getAllClaims) to it as createClaim already uses PostMapping